resources/Materials/Examples/StandardSurface/standard_surface_brass_tiled.mtlx

```diff
@@ -16,12 +16,11 @@
   <material name="Tiled_Brass">
     <shaderref name="SR_brass1" node="standard_surface">
       <bindinput name="base" type="float" value="1" />
+      <bindinput name="base_color" type="color3" nodegraph="NG_brass1" output="out_color" />
       <bindinput name="specular" type="float" value="0" />
+      <bindinput name="specular_color" type="color3" value="1.0, 1.0, 1.0" />
       <bindinput name="specular_roughness" type="float" nodegraph="NG_brass1" output="out_roughness" />
       <bindinput name="metalness" type="float" value="1" />
-      <bindinput name="coat" type="float" value="1" />
```

Comment:
Fix up parameters for a metal. Coat is not meant to be used for this.

Reply:
If I'm remembering correctly, this unusual usage of "coat over metal" to represent brass was borrowed from one of the metals in the public Standard Surface library for Maya.  Not that we need to follow this convention in our material examples, but I believe that's the origin.